Brought readme up to date.

hovi opened the pull request.

PremekPaska commented on README.md:
````diff
@@ -1,69 +1,50 @@
 
-**Work in progress notice.** Keep in mind that the work here is still in progress. Things may look ugly while we are working on them, and things may change.
+## Clone repo:
 
-**What is the demo?** A set of JUnit test classes demonstrating how iv4xr test agents are used to implement a number of testing tasks for the game _Lab Recruits_. These classes are located in `src/test/java/agents/demo`. You can simply run them, or modify them yourself. When you run them you don't usually see anything because the tests run pretty fast. You can insert pause-points yourself (e.g. u ...
+```
+git clone git@github.com:iv4xr-project/iv4xrDemo-space-engineers.git
+```
````
This is kind of obvious. Why is it here? Is it just for easy copy-paste to cmd line? Am I missing something?

hovi replied:
Exactly, easy copy-paste.

PremekPaska replied:
Hmm, I find it confusing. You have the link in the corner of every repository on GitHub. Also many people use Gui tools nowadays.

hovi replied:
That's the thing. You use GUI tools, then you find it useless. I on the other hand use command line a lot and I always appreciate when it is there. I can get the point where it takes space and we want to keep readme file smart, but how exactly is this line confusing for anyone?

PremekPaska commented on README.md:
```diff
@@ -1,69 +1,50 @@
-Stijn Hinlopen,
-Tom Tanis.
-**Others:** Wishnu Prasetya, Naraenda Prasetya.
+* Right-click .feature file in Jetbrains IDEA (in src/jvmTest/resources/features) and select "Run". 
```
Please list the plugins required to run Cucumber.

* Gherkin
* Cucumber for Java

Plus link [here](https://www.jetbrains.com/help/idea/enabling-cucumber-support-in-project.html#cucumber-plugin).

PremekPaska reviewed (commented):
Please add a section about Kotlin. Some points:
- this demo is in Kotlin (or "mostly", after we add some Java examples?)
- Kotlin is a JVM language by JetBrains fully interoperable with java, i.e. can be used seamlessly from Java
- What you need to compile / run it

PremekPaska committed:
Brought readme up to date.

Introduction update, small fixes.


Updated readme file based on comments.

Small readme fixes.
